DocForMe/Otus05/README.md

```diff
@@ -0,0 +1,169 @@
+
+11. Create firewall rule
+    * use your IP address
+    * or allow all IP 0.0.0.0/0
+    * set rule targets "All instances in the network" ![Targets](2022-08-30-06-16-11.png)
+    * set protocol tcp port 22
+    * save rule
+    * QUESTION: How to specify ip address?
+
+12. Create instance VM
+    * Hostname: Bastion
+    * Region: us-central
+    * Machine Type: e2-micro ![setup list](2022-08-29-07-25-06.png)
+    * Boot disk: Ubuntu - ver. 20.04 (because 22.04 not work with MongoDB at the time of creating this instruction)  ![OS Version](2022-08-29-07-26-32.png)
+    * Allow HTTP&HTTPS traffic ![HTTP(s) Allow Settings](2022-08-30-05-56-40.png)
+    * Add network tags `bastion`
+
+13. Open Advanced option and setup networking like on screenshot![NetworkSettings](2022-08-29-07-34-10.png)
+
+14. Create VM![VM created](2022-08-29-07-46-31.png)
+    * Check that your VM appeared in the Firewall rule you created ![firewall list](2022-08-30-06-24-02.png)
+    * If appeared check ssh connection ![ssh-check](2022-08-30-06-20-19.png) you should see the VM terminal
+
+15. try to connect to bastion with terminal linux and using early created ssh key
+    * `ssh -i ~/.ssh/appuser appuser@externalIpAddrBastion` ![log](2022-08-30-06-28-57.png)
+
+## Second internal-host
+
+1. Create second VM without external network ![external settings](2022-08-30-06-35-23.png)
+
+2. Check ![internal host](2022-08-30-06-38-31.png)
+
+3. try to connect to bastion (Item 9), and then in bastion terminal try to connect to internal-host
+    * if you setup firewall allow only your IP, you need add internal IP your bastion setting in firewall rule or create new firewall rule with this IP address
+    * `$ ssh internalIp` ![internal Ip](2022-08-30-06-43-55.png)
+    * if all right you see "permission denied" ![permission](2022-08-30-06-45-10.png)
+
+4. Setup Bastion host for direct connection internalnetwork GCP
+    * in localhost (your PC/laptop) setup SSH Forwarding
+      * ```$ eval `ssh-agent```
+      * `$ ssh-add -L` (check ssh-agent list)
+      * `$ ssh-add ~/.ssh/appuser` (add to ssh-agent list your ssh private key)![steps](2022-08-30-06-58-46.png)
+    * try to connection to internal-host via bastion
+      * `$ ssh -i ~/.ssh/appuser -A appuser@bastionIp`
+      * `$ ssh internal-host-ip` ![expected terminal log](2022-08-30-07-04-03.png)
+    * make sure that you are on the right host![check host](2022-08-30-07-06-38.png)
+
+5. one-command connection to the internal host ssh -J appuser@bastionIp appuser@internalHostIp
+
+6. add in `~/.ssh/appuser` on your localhost next:
+   * before edit, create backup this file
+   * change IPAddres to yours
+   ![ssh-config](2022-09-01-12-03-37.png)
+  
+7. save changes and restart terminal to make the function avalible
+
+8. try to connect use function-alias `internal` ![aliasConnection](2022-09-01-12-06-54.png)
+
+## Create VPN sever for GCP
+
+1. Login to bastion via ssh
+2. Create bash script file: `touch setupvpn.sh`
```

Comment:
Travis CI build fails because your  `setupvpn.sh` script is empty so you must move this code from README to `setupvpn.sh` script